cli/tools/upgrade.rs

```diff
@@ -725,6 +890,7 @@ fn get_latest_version_url(
     ReleaseChannel::Canary => {
       Cow::Owned(format!("canary-{target_tuple}-latest.txt"))
     }
+    ReleaseChannel::Rc => Cow::Borrowed("release-2-rc.txt"),
```

Comment:
Since this PR provides a generic solution to RCs it's probably best to rename it to `release-rc.txt`